eslint.config.js

```diff
@@ -0,0 +1,11 @@
+// eslint.config.js
+import js from "@eslint/js";
+
+export default [
+  js.configs.recommended,
+  {
+    rules: {
+      "no-unused-vars": "warn"
+    }
+  }
+];
```

Comment:
This eslint configuration doesn't seem to be equivalent to the previous?

Reply:
hmm true.. tho i haven't encountered eslint working properly in my IDE so figured i shouldn't spend too much time on it.. tho i can look it over a bit more in detail

Reply:
I would try and make sure the rules are all the same, the old config had a fair amount of customisation

Reply:
Didn't update eslint in the end